go/ct/spc/specification.go

```diff
@@ -131,6 +131,18 @@ func getAllRules() []Rule {
 		}),
 	})
 
+	// --- No Code ---
+
+	rules = append(rules, Rule{
```

Comment:
Adding this rule is likely introducing a contradiction between rules.

The way the problem this PR is fixing was discovered is through the CT. This means a state with the empty code was generated, passed through the specification, a rule was found, and this rule states that the effect should be that the execution stopped. By adding this rule you provide a contradicting definition to the rule that previously handled this case, which is invalid.

You should create a state with the empty code and ask the specification which rules are responsible for this state. Then this conflict must be resolved between the existing rules an this new rule.